### DISCUSSION TITLE
Old Forum Articles

### DISCUSSION THREAD

**Discussion by mrpastewart** [leokhoa/laragon#691] [General]
Hello

Do you have plans to move the old articles over to the mew discussions boards? Several of these articles are linked from inside Laragon itself and are no longer available for new users to read as they discover the platform. At minimum the articles that are referenced from the software seem relevant to move over as they were linked from the menu for a good reason.

**Comment by josealgardataborges** [comment]
Try using the WayBackMachine, some content is still browsable: https://web.archive.org/web/20230201000000*/https://forum.laragon.org/

**Comment by kstingel** [comment]
I've started work on recovering 'lost' documenation, check the [`Wiki`](https://github.com/leokhoa/laragon/wiki) for recovered content

- [Procfile - What is it?](https://github.com/leokhoa/laragon/wiki/Laragon-feature:-Procfile-%E2%80%90-What-is-it%3F)
